ambry-store/src/main/java/com.github.ambry.store/BlobStoreCompactor.java

```diff
@@ -120,6 +126,8 @@ public boolean accept(File dir, String name) {
     this.sessionId = sessionId;
     this.incarnationId = incarnationId;
     fixStateIfRequired();
+    this.bundleReadBuffer =
+        ByteBuffer.allocate(Math.max(config.storeCleanupOperationsBytesPerSec * 2, 10 * 1024 * 1024));
```

Comment:
should we make this a config?

Reply:
You mean another individual config other than `storeCleanupOperationsBytesPerSec `? I prefer to have a individual config for the bufferSize.

Reply:
yeah - we still have the `max()` code is what I'm thinking but have a "compaction.min.buffer.size" instead of harcoding to 10.